wiki/layout/subpage-bottom-nav.html

```diff
@@ -7,14 +7,8 @@
 <div class="flex flex-wrap w-full lg:px-20 xl:px-60 justify-center mb-8">
   <template x-for="(sp, index) in subpages" :key="sp.key">
     <template x-if="sp.img != ''">
-      <div
```

Comment:
Can you undo this removal? Always make sure to start a new branch from main and don't continue from your other feature branches. Then I can review them separately!

Reply:
Sorry, it should be undone now.